runtime/commonTest/src/kotlinx/serialization/features/SchemaTest.kt

```diff
@@ -13,6 +13,11 @@ import kotlin.test.*
 
 class SchemaTest {
 
+    enum class SampleEnum { OptionA, OptionB, OptionC }
```

Comment:
I'd prefer leave these classes in a separate file in case we want to reuse them in new tests

Reply:
It's always easy to extract them later.
Now count of serializable classes available in tests (and used only in a single test) large enough to get lost